sites/public/pages/finder.tsx

```diff
@@ -0,0 +1,213 @@
+    )
+    setFormData(formCopy)
+    setCurrentStep(currentStep + 1)
+  }
+  const previousQuestion = () => {
+    const userSelections = getValues()[formData[currentStep]["fieldGroupName"]]
+    const formCopy = { ...formData }
+    formCopy[currentStep]["fields"].forEach(
+      (field) => (field["selected"] = userSelections.includes(field))
+    )
+    setFormData(formCopy)
+    setCurrentStep(currentStep - 1)
+  }
+
+  return (
+    <Layout>
+      <Form onSubmit={handleSubmit(onSubmit)} className="bg-gray-300 border-t border-gray-450">
+        <div className="md:mb-8 mt-8 mx-auto max-w-5xl">
+          {ProgressHeader()}
+          <FormCard>
+            {formData && (
+              <>
+                <div className="px-10 md:px-20 pt-6 md:pt-12 ">
+                  <div className="">
+                    <div className="text-3xl pb-4">
+                      {t(`finder.${activeQuestion?.fieldGroupName}.question`)}
+                    </div>
+                    <div className="pb-8 border-b border-gray-450">
+                      {t("finder.questionSubtitle")}
+                    </div>
+                  </div>
+                  <div className="py-8">
+                    <p className="pb-4">{t("finder.multiselectHelper")}</p>
+                    <div className="finder-grid">
+                      {activeQuestion?.fields?.map((field) => {
+                        return (
+                          <div
+                            className="finder-grid__field"
+                            key={FrontendListingFilterStateKeys[field.label]}
+                          >
+                            <Field
+                              name="bedRoomSize"
+                              register={register}
+                              id={FrontendListingFilterStateKeys[field.label]}
+                              label={t(`listingFilters.bedroomsOptions.${field.translation}`)}
+                              key={FrontendListingFilterStateKeys[field.label]}
+                              type="checkbox"
+                              inputProps={{
+                                value: FrontendListingFilterStateKeys[field.label],
+                                defaultChecked: field.selected,
+                              }}
+                              bordered
+                            />
+                          </div>
+                        )
+                      })}
+                    </div>
+                  </div>
+                </div>
+
+                <div className="bg-gray-300 flex flex-row-reverse justify-between py-8 px-20">
+                  {currentStep === Object.keys(formData).length ? (
+                    <Button type="submit" styleType={AppearanceStyleType.primary}>
+                      {t("t.submit")}
+                    </Button>
+                  ) : (
+                    <Button
+                      type="button"
+                      onClick={() => nextQuestion()}
```

Comment:
You don't need to create a new arrow function here. You could just do `onClick={nextQuestion}`. Same for the previous question